manifests/env.pp

```diff
@@ -24,7 +24,7 @@
   String[1]                      $version,
   Enum['present', 'absent']      $ensure           = 'present',
   Array[String[1]]               $makeopts         = [],
-  Optional[Stdlib::Absolutepath] $makefile         = undef,
+  Optional[String[1]]            $makefile         = undef,
```

Comment:
I'm afraid someone will update it to Stdlib::Absolutepath in the future if we don't document it properly. maybe add a comment at the end?
```suggestion
  Optional[String[1]]            $makefile         = undef, # String and not Stdlib::Absolutepath because relative paths are supported as well
```

Reply:
I set it to `puppet:///moduldes/profile/poudriere/default-make.conf` in the CI test suite for this purpose :smiley: .  Do you think we can skip this comment?

Reply:
ah good idea, I think that's fine.